src/server/_params.py

```diff
@@ -425,44 +425,21 @@ def extract_dates(key: Union[str, Sequence[str]]) -> Optional[TimeValues]:
         return None
     values: TimeValues = []
 
-    def push_range(first: str, last: str):
-        first_d = parse_date(first)
-        last_d = parse_date(last)
-        if first_d == last_d:
-            # the first and last numbers are the same, just treat it as a singe value
-            return first_d
-        if last_d > first_d:
-            # add the range as an array
-            return (first_d, last_d)
-        # the range is inverted, this is an error
-        raise ValidationFailedException(f"{key}: the given range is inverted")
-
     for part in parts:
-        if "-" not in part and ":" not in part:
-            # YYYYMMDD
-            values.append(parse_date(part))
+        if "*" in part:
+            values.append(part)
             continue
```

Comment:
```suggestion
        if "*" == part:
            return ["*"]
```
i think the `==` part is required, otherwise itll match a `part` like "x*y"

Reply:
...the `return` suggestion seems like a good way to short-circuit the rest of the argument processing -- if we have a wildcard to allow all dates, the other dates should be redundant

Reply:
Yeah, good point!